engine/classes/Elgg/Database/RelationshipsTable.php

```diff
@@ -117,11 +117,8 @@ function add($guid_one, $relationship, $guid_two) {
 		if ($id !== false) {
 			$obj = get_relationship($id);
 	
-			// this event has been deprecated in 1.9. Use 'create', 'relationship'
-			$result_old = _elgg_services()->events->trigger('create', $relationship, $obj);
-	
 			$result = _elgg_services()->events->trigger('create', 'relationship', $obj);
```

Comment:
I.e. why do views get `view,$view` but relationships have to settle for `create,releationship`?